README.md

```diff
@@ -298,6 +298,47 @@ by its ID with the `custom.` prefix:
 />
 ```
 
+### Custom Queries within catalog-info.yaml file of the Backstage Entity
+
+You can also register your custom queries in the catalog-info.yaml file.
+
+```yaml
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: demo-backstage
+  description: Backstage Demo instance.
+  annotations:
+    backstage.io/kubernetes-id: kubernetescustom
+spec:
+  type: website
+  owner: user:default/mjakl
+  lifecycle: experimental
+  system: integrations
+  queries:
+    - name: Fetch all Davis events 1
+      query: >
+        fetch events | filter event.kind == "DAVIS_EVENT" | fields event.kind,
+        timestamp
+```
+
+As mentioned before, queries can contain placeholders. In the catalog-info.yaml
+file, the placeholders are prefixed with a single `$`. For example:
```

Comment:
Instead of providing examples, please link to: https://github.com/Dynatrace/backstage-plugin/blob/eac6adfe0c25fc7a4e5b0b7d05d5dc83464f3652/README.md#custom-queries

The text should be: `Please find the supported placeholders listed [here](https://github.com/Dynatrace/backstage-plugin/blob/eac6adfe0c25fc7a4e5b0b7d05d5dc83464f3652/README.md#custom-queries).`